vnext/Microsoft.ReactNative/Fabric/Composition/UiaHelpers.cpp

```diff
@@ -214,4 +214,24 @@ ExpandCollapseState GetExpandCollapseState(const bool &expanded) noexcept {
   }
 }
 
+void AddSelectionItemsToContainer(CompositionDynamicAutomationProvider *provider) noexcept {
+  IRawElementProviderSimple *selectionContainer;
+  provider->get_SelectionContainer(&selectionContainer);
```

Comment:
No I don't believe it's releasing memory to populate the variable. See comment below. Unfortunately I can't change the type of get_SelectionContainer because it's set by UIA